File: sumprimes/sumprimes.py
def check_variables(value):
  primes=[2,3,5,7]
  
  if value in primes:
    return value
  else:
    return 0


def sum_variables(value):
  total=0
  
  while(value >0):
   total=check_variables(value)+total
   value=value-1
  return total
print(sum_variables(10)) 

File: sumprimes/sumprimes.java
public class SumPrimes{

   /* calculating the sum of primes
    */

   public static void main(String []args) {
     sumPrimeValues(12);
      System.out.println("Hello World"); // prints Hello World
   }
   private static int checkPrimeValue(int value){
    int[] myList = {2, 3, 5, 7};

      for (int i = 0; i < myList.length; i++) {
         if(myList[i] ==value){
          return value;
         }
         
         if( (value % myList[i])==0){
           return value;
         } 
      }
    return 0;
   }
   private static void sumPrimeValues(int value){
   
     int total=0;
   
     while(value>0) {
      
       total =total+checkPrimeValue(value);
       
       value--;
     }
     System.out.println("The total is "+total);
    }  
}
